l5kit/l5kit/kinematic/ackerman_perturbation.py

```diff
@@ -76,6 +68,23 @@ def perturb(self, history_frames: np.ndarray, future_frames: np.ndarray) -> Tupl
         curr_frame_idx = num_history_frames - 1
         displacements = np.linalg.norm(np.diff(trajectory[curr_frame_idx:, :2], axis=0), axis=1)
 
+        if num_history_frames > 1:
```

Comment:
IMHO we should explain this is the function docs, that it has different behavior, depending on availability of history frames.